packages/financial-templates-lib/src/price-feed/CreatePriceFeed.js

```diff
@@ -165,6 +166,31 @@ async function createPriceFeed(logger, web3, networker, getTime, config) {
       config.denominatorPriceFeed && (await _createMedianizerPriceFeed(config.denominatorPriceFeed));
 
     return new BasketSpreadPriceFeed(web3, logger, baselinePriceFeeds, experimentalPriceFeeds, denominatorPriceFeed);
+  } else if (config.type === "tradermade") {
+    const requiredFields = ["pair", "apiKey", "minTimeBetweenUpdates"];
+
+    if (isMissingField(config, requiredFields, logger)) {
+      return null;
+    }
+
+    logger.debug({
+      at: "createPriceFeed",
+      message: "Creating TraderMadePriceFeed",
+      config
+    });
+
+    return new TraderMadePriceFeed(
+      logger,
+      web3,
+      config.apiKey,
+      config.pair,
+      config.minuteLookback,
+      config.hourlyLookback,
+      networker,
+      getTime,
+      config.minTimeBetweenUpdates,
+      config.priceFeedDecimals // Defaults to 18 unless supplied. Informs how the feed should be scaled to match a DVM response.
```

Comment:
Should the user be able to also configure the ohlcPeriod?
```suggestion
      config.priceFeedDecimals, // Defaults to 18 unless supplied. Informs how the feed should be scaled to match a DVM response.
      config.ohlcPeriod
```

Reply:
I see, maybe the backup_branch record an old version..., thanks for pointing this